website/docs/reference/widgets/list.md

```diff
@@ -121,45 +114,26 @@ To better understand how to bind data to a list widget, let’s take an example
 
 You’ll see that in the JSON snippet below, there is a collection of books with details like `bookId`, `bookName`, `bookImage`, `category`, `author`, `publishedDate`, and `price`.
 
-```
+```json
```

Comment:
If we are updating the object schema, we should also update the line above! cc @ashit-rath

Reply:
Haven't updated the object here, just removed empty lines and added the `json` word for better formatting.